### PR TITLE
Update Korean translation

### PR DESCRIPTION
**Pull request by yous**
- Fix Korean grammar introduced in #78.
- Translate string-length to Korean.

@marocchino, @taggon, @xarus01, @hibiyasleep, @nundefined, @outsideris Review please.

@sethvincent Please wait to merge until the review.

### CODE REVIEW

File: problems/functions/problem_ko.md [hunk 1]
```diff
@@ -18,13 +18,14 @@ function example (x) {
...
-위의 예제는 `example` 함수가 숫자를 인자 (입력)로 받아 그 숫자에 2를 곱한 값을 반환합니다.
+위의 예제는 `example` 함수가 숫자를 인자(입력)로 받아 그 숫자에 2를 곱한 값을 반환합니다.
 
 ## 도전 과제
 
 `functions.js`라는 파일을 만듭니다.
 
-그 파일에서 문자열이어야 하는 `food`라는 이름의 인자를 받는 `eat`이라는 파일을 선언합니다.
+그 파일에서 `food`를 인자로 받는 `eat` 함수를 선언합니다. `food`는 문자열이어야
+합니다.
```
Comment (marocchino) [inline problems/functions/problem_ko.md]: This text will print in terminal. Don't enter middle of sentence.

File: problems/objects/problem_ko.md [hunk 2]
```diff
@@ -2,8 +2,8 @@
 
 # 객체
 
-객체는 배열과 비슷한 값의 목록입니다. 배열과 다른 점은 정수대신 키를 사용해 값을
-확인하는 점입니다.
+객체는 배열과 비슷한 값의 목록입니다. 배열과 다른 점은 정수 대신 키를 사용해
```
Comment (marocchino) [inline problems/objects/problem_ko.md]: same here.

File: problems/string-length/problem_ko.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+var example = 'example string';
+example.length
+```
+
+# 주의
+
+`example`과 `length` 사이에 마침표가 있는 것을 확인하세요.
+
+위의 코드는 문자열 안에 있는 전체 문자의 **수**를 반환합니다.
+The above code will return a **number** for the total number of characters in the string.
```
Comment (marocchino) [inline problems/string-length/problem_ko.md]: This line should deleted.

**Comment by yous** [comment]
Applied reviews.

**Comment by marocchino** [comment]
:+1: for me.

**Comment by SomeoneWeird** [comment]
Good for merge?

**Comment by yous** [comment]
Yes, please. :smiley_cat:

**Pull request merged**